changelogs/unreleased/gh-7797-untrusted-input-validation.md

```diff
@@ -1,3 +1,5 @@
 ## bugfix/core
 
 * Added boundary checking for getenv() return values (gh-7797).
+* Introduced `os.getenv_limited` helper, exposed `os.PATH_MAX` to lua and
+  started using them in internal code instead of `os.getenv` (gh-7797).
```

Comment:
It seems, it was renamed to `getenv_safe()` at some point, but the name was not updated here.